bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java

```diff
@@ -104,6 +112,19 @@ public Builder setShortRpcTimeoutMs(int shortRpcTimeoutMs) {
       return this;
     }
 
+    /**
+     * The amount of milliseconds to wait before issuing a client side timeout for short RPCs.
+     *
+     * @param shortRpcAttemptTimeoutMs timeout value in milliseconds.
+     * @return a {@link Builder} object, for chaining
+     */
+    public Builder setShortRpcAttemptTimeoutMs(int shortRpcAttemptTimeoutMs) {
```

Comment:
Can you annotate all of the new methods with `@BetaApi`?

Reply:
Done